docs/content/config/advanced/auth-ldap.md

```diff
@@ -126,6 +126,62 @@ To enable LDAP over StartTLS (on port 389), you need to set the following enviro
 - SASLAUTHD_LDAP_START_TLS=yes
 ```
 
+## Active Directory Configurations (Tested with Samba4 AD Implementation)
+
+In addition to LDAP explanation above, when Docker Mailserver is intended to be used with Active Directory (or the equivelant implementations like Samba4 AD DC) the following points should be taken into consideration:
+
+- Samba4 Active Directory requires a **secure connection** to the domain controller (DC), either via SSL/TLS (LDAPS) or via StartTLS.
+- The username equivenlant in Active Directory is `sAMAccountName`.
+- `proxyAddresses` can be used to store email aliases of single users. The convention is to prefix the email aliases with `smtp:` (e.g., smtp:some.name@DOMAIN.COM).
+- Active Directory is used typically not only as LDAP Directory storage, but also as a domain controller, i.e., it will do many things including authenticating users. Mixing Linux and Windows clients requires the usage of [RFC2307 attributes](https://wiki.samba.org/index.php/Administer_Unix_Attributes_in_AD_using_samba-tool_and_ldb-tools), namely `uidNumber`, `gidNumber` instead of the typical `uid`. Assigning different owner to email folders can also be done in this approach, nevertheless [there is a bug at the moment in Docker mailserver that overwrites all permissions](https://github.com/docker-mailserver/docker-mailserver/pull/2256?notification_referrer_id=NT_kwDOAOOOe7MyNTcxNjg5OTQxOjE0OTEzMTQ3#pullrequestreview-811816197) when starting the container. So either a manual fix is necessary now, or a temporary workaround to use a hardcoded `ldap:uidNumber` equals to 5000 until this issue is fixed.
+- To deliver the emails to different members of Active Directory **Distribution Group** (in other simple words, mailing lists), editing *ldap-groups.cf* to add `leaf_result_attribute = mail` and `special_result_attribute = member` using a [manually written user-patch](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/override-defaults/user-patches/) is necessary. This can be achieved as simple as this:
+
+The configuration shown to get Distribution Group to work is from [here](https://doc.zarafa.com/trunk/Administrator_Manual/en-US/html/_MTAIntegration.html) and [here](https://kb.kopano.io/display/WIKI/Postfix).
+
+```
+# user-pachtes.sh
+
+...
+echo "leaf_result_attribute = mail" >> /etc/postfix/ldap-groups.cf
+echo "special_result_attribute = member" >> /etc/postfix/ldap-groups.cf
+...
+```
+
+- In */etc/ldap/ldap.conf*, if the TLS_REQCERT is demand/hard (which is the default), the CA certificate used to sign the certificate of the LDAP server must be registered as a trusted CA. The easiest way to it is to mount the ca.crt file and register it in a user-patch script:
```

Comment:
```suggestion
- In `/etc/ldap/ldap.conf`, if the `TLS_REQCERT` is `demand` / `hard` (default), the CA certificate used to verify the LDAP server certificate must be recognized as a trusted CA. This can be done by volume mounting the `ca.crt` file and updating the trust store via a `user-patches.sh` script:
```

Reply:
Actually here the CA used to verify is originally the same one used to sign. I would keep it "sign", but there not a big difference.